docs/configurations.rst

```diff
@@ -28,18 +30,30 @@ Cheyenne
 
    jobqueue:
      pbs:
-       cores: 36
-       memory: 108GB
-       processes: 4
+       name: dask-worker
+       cores: 36                   # Total number of cores per job
+       memory: '109 GB'            # Total amount of memory per job
+       processes: 9                # Number of Python processes per job
+       interface: ib0              # Network interface to use like eth0 or ib0
+
+       queue: premium
+       project: PXYZ123
+       walltime: '00:30:00'
+       resource-spec: select=1:ncpus=36:mem=109GB
+
+     slurm:
+       name: dask-worker
+
+       # Dask worker options
+       cores: 1                    # Total number of cores per job
+       memory: '25 GB'             # Total amount of memory per job
+       processes: 1                # Number of Python processes per job
 
        interface: ib0
-       local-directory: $TMPDIR
 
-       queue: regular
-       project: null  # TODO, change me
+       project: PXYZ123
```

Comment:
I anticipate that people who don't pay attention won't change this.  I slightly prefer leaving this as `null` so that they get an appropriate message about it.